medusa/search/manual.py

```diff
@@ -34,14 +36,19 @@
 SEARCH_STATUS_SEARCHING = 'searching'
 
 
+# Get key names from `common.Quality` class (for the quality values and any-sets)
+# Copied from `medusa/server/api/v2/config.py` @ `DataGenerator.data_consts`
+quality_key_map = {
+    value: text_type(key).lower().replace('_', '')
+    for (key, value)
+    in iteritems(Quality.__dict__)
+    if isinstance(value, int) and not key.startswith('__')
+}
+
+
 def get_quality_class(ep_obj):
```

Comment:
What about putting this in Quality? I don't see a strong reason to keep this in manual search.

Reply:
I don't want this at all, but I still need it for [the AJAX episode search](https://github.com/pymedusa/Medusa/blob/d2f971afa85b7c438f47c32e28c60e51c6125152/themes-default/slim/static/js/ajax-episode-search.js#L55-L62).
(😅 everything is dependencies upon dependencies, that's why I'm trying to convert the smaller components to Vue first)
Anyway, I don't see us needing it anywhere else in the future. I believe that I removed everything related to qualities' CSS classes from Python...